core/system/src/svc.c

```diff
@@ -127,7 +127,7 @@ void UVISOR_NAKED SVCall_IRQn_Handler(void)
         ".word  __svc_not_implemented\n"
     "jump_table_unpriv_end:\n"
 
-    ".thumb_func\n"                            // needed for correct referencing
+        ".thumb_func\n"                            // needed for correct referencing
```

Comment:
Here for all re-indented directives I would re-indent also the comment to make it aligned with the others (also for other directives below this one)